setup.cfg

```diff
@@ -30,6 +30,7 @@ test =
     pytest
     pytest-cov
     pandas<2
+    pyarrow~=8.0.0
```

Comment:
is this deliberate? can't tell what the other changes need this for

Reply:
we don't need the version pin actually, the main reason we need the dependency is to test the parquet serde.